src/legacy/core_plugins/region_map/public/region_map_visualization.js

```diff
@@ -164,7 +164,8 @@ export function createRegionMapVisualization({ serviceSettings, $injector, uiSet
       }
 
       this._choroplethLayer.on('select', event => {
-        const rowIndex = this._chartData.rows.findIndex(row => row[0] === event);
```

Comment:
Okay, so it looks like `rowIndex` was always returning `-1` before, and there used to be a special case for this.